query/group/azuread_group_not_public.sql

```diff
@@ -0,0 +1,15 @@
+select
+  -- Required Columns
+  id as resource,
+  case
+    when visibility = 'Public' then 'alarm'
+    else 'ok'
+  end status,
+  case
+    when visibility = 'Public' then display_name || ' is public.'
```

Comment:
```suggestion
    when visibility = 'Public' then title || ' is public.'
```

We usually use `title` as part of reasons since it's easier to remember to just use `title`, and the titles are usually descriptive enough to let people know what each row represents. Does `title` show the same value as `display_name` for this table (if so, we should switch, and if not, we should look at title)?